Classes/Repository/CacheMd5paramsRepository.php

```diff
@@ -0,0 +1,105 @@
+<?php
+declare(strict_types=1);
+
+/*
+ * This file is part of the TYPO3 CMS project.
+ *
+ * It is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License, either version 2
+ * of the License, or any later version.
+ *
+ * For the full copyright and license information, please read the
+ * LICENSE.txt file that was distributed with this source code.
+ *
+ * The TYPO3 project - inspiring people to share!
+ */
+
+namespace FoT3\Rdct\Repository;
+
+use TYPO3\CMS\Core\Database\Connection;
+use TYPO3\CMS\Core\Database\ConnectionPool;
+use TYPO3\CMS\Core\Database\Query\QueryBuilder;
+use TYPO3\CMS\Core\Utility\GeneralUtility;
+
+class CacheMd5paramsRepository
+{
+    protected string $table = 'cache_md5params';
+
+    public function getConnectionPool(): ConnectionPool
```

Comment:
These methods do not need to be public I guess.

Reply:
Thx, I have changed to protected.